src/licensedcode/data/licenses/ecos.yml

```diff
@@ -5,12 +5,11 @@ name: GPL 2.0 or later with eCos Exception
 category: Copyleft Limited
 owner: eCos
 homepage_url: http://ecos.sourceware.org/ecos-license/
-notes: Per SPDX.org, this is really GPL v2 or later + an exception
```

Comment:
May be this can be added to the bottom on the remaining notes field?

Reply:
Merged.